src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/article/ArticleNavigationComponentObject.java

```diff
@@ -0,0 +1,28 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.article;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * @ownership Content X-Wing Wikia
+ *
+ * Represents the wikia navigation present on an article page
```

Comment:
We need a description here. The repository suffers from poor documentation. It's often hard to guess what a component object represents